src/interfaces/UBA.ts

```diff
@@ -11,9 +11,17 @@ export type UbaRunningRequest = {
 };
 
 export const isUbaInflow = (flow: UbaFlow): flow is UbaInflow => {
-  return (flow as UbaInflow).depositId !== undefined;
+  return (flow as UbaInflow).quoteTimestamp !== undefined;
 };
 
 export const isUbaOutflow = (flow: UbaFlow): flow is UbaOutflow => {
-  return !isUbaInflow(flow);
+  return !isUbaInflow(flow) && (outflowIsFill(flow) || outflowIsRefund(flow));
+};
+
+export const outflowIsFill = (outflow: UbaOutflow): outflow is FillWithBlock => {
+  return (outflow as FillWithBlock).isSlowRelay !== undefined;
+};
+
+export const outflowIsRefund = (outflow: UbaOutflow): outflow is RefundRequestWithBlock => {
+  return (outflow as RefundRequestWithBlock).fillBlock !== undefined;
 };
```

Comment:
Nit: now that we have a few of these predicates, would it make sense to place these in their own file?